src/marinade.ts

```diff
@@ -255,7 +255,7 @@ export class Marinade {
     }
 
     const validatorRecords = await marinadeState.getValidatorRecords()
```

Comment:
please destructure here too, so we avoid 
`validatorRecords.validatorRecords.findIndex` in line 258

Reply:
I updated it. Thank you!